app/controllers/container_controller.rb

```diff
@@ -95,6 +95,7 @@ def show_list
   end
 
   def explorer
+    process_show_list
```

Comment:
about 2: Other views are using the process_show_list in this case. It also clears previous breadcrumbs and does other things.